go/utils/intoto/slsa_zero_two_converter.go

```diff
@@ -0,0 +1,131 @@
+package intoto
+
+import (
+	"fmt"
+	"time"
+
+	pb "github.com/grafeas/grafeas/proto/v1/grafeas_go_proto"
+	intoto "github.com/in-toto/in-toto-golang/in_toto"
+	"google.golang.org/protobuf/types/known/structpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// SLSAZeroTwoToProto function converts the intoto statement with SLSA 0.2 predicate to the
+// grafeas proto InTotoStatement where slsa 0.2 predicate is oneof the predicate types
+func SLSAZeroTwoToProto(in *intoto.ProvenanceStatement) (*pb.InTotoStatement, error) {
```

Comment:
bikeshed: 
```suggestion
func ToSLSAZeroTwoProto(in *intoto.ProvenanceStatement) (*pb.InTotoStatement, error) {
```

`SLSAZeroTwoToProto` can read as `SLSAZeroTwo` -> `Proto`, which isn't what this is doing.

Reply:
aha. Let me just use `ToProto` then.